src/Common/Database/QueryUtils.php

```diff
@@ -14,6 +14,24 @@
 
 class QueryUtils
 {
+    /**
+     * Function that will return an array listing
+     * of columns that exist in a table.
+     *
+     * @param   string  $table sql table
+     * @return  array
+     */
+    public static function listTableFields($table)
+    {
+        $sql = "SHOW COLUMNS FROM " . \add_escape_custom($table);
```

Comment:
in regards to escaping, need single quotes around things for it to work (although I am not sure if able to place quotes around this type of query element)

Reply:
Fied.